<launch>  
  <!-- so the logging print the node name -->
  <env name="ROSCONSOLE_FORMAT" value="[${severity}] [${node}]: ${message}" />
  <env name="ROSCONSOLE_CONFIG_FILE" value="$(find towr_ros)/config/towr.conf"/>
  
  <!--  load the ros parameters  -->
  <include file="$(find towr_ros)/launch/towr_params.launch"></include>
  
  <!-- for commanding goal positions and tasks -->
  <node 
    name="keyboard" 
    pkg="keyboard" 
    type="keyboard" 
    output="screen" 
    cwd="node" >  
  </node>
  
  
  <!-- for translating keyboard messages to TOWR commands -->
  <node 
    name="user_interface_node" 
    pkg="towr_ros" 
    type="user_interface_node" 
    output="screen" 
    cwd="node" >  
  </node>
   
  
  <node 
    name="nlp_optimizer_node" 
    pkg="towr_ros" 
    type="nlp_optimizer_node" 
    output="screen">
  </node>
    
</launch>
